vscode/src/chat/context/constants.ts

```diff
@@ -4,7 +4,7 @@ export const SYMBOL_HELP_LABEL = 'Search for a symbol to include...'
 export const NO_SYMBOL_MATCHES_LABEL = 'No symbols found'
 export const NO_FILE_MATCHES_LABEL = 'No files found'
 export const NO_SYMBOL_MATCHES_HELP_LABEL = ' (language extensions may be loading)'
-export const FILE_RANGE_TOOLTIP_LABEL = 'Type a line range to include, e.g. 5-10...'
+export const FILE_RANGE_TOOLTIP_LABEL = 'File too large. Type a line range like 5-10...'
 export const LARGE_FILE_WARNING_LABEL =
     'File too large. Add line range with : or use @# to choose a symbol'
```

Comment:
wdyt of removing `@# to choose a symbol`?

```suggestion
    'File too large. Type : to select a subset of lines from the file.'
```

 I don't think it's reasonable to ask them to hit backspace and do a global symbol search. The original label was "Type # to choose a symbol" and it would then show only symbols from the file.

Reply:
This is before they've necessarily typed much, though. I don't think it's unreasonable. Neither is great, though.